### ISSUE TITLE
obj.IsRepeatable undefined (type *introspection.Directive has no field or method IsRepeatable)

### ISSUE THREAD

**Issue by khanakia** [99designs/gqlgen#1544]
### What happened?
I am getting obj.IsRepeatable error when download using go mod download but when i see code here `https://github.com/99designs/gqlgen/blob/master/graphql/introspection/introspection.go` i see you have `IsRepeatable bool` but when i check the struct downloaded using `go mod` i do not see that variable check screenshot

<img width="931" alt="Screenshot 2021-06-06 at 8 56 01 AM" src="https://user-images.githubusercontent.com/2174170/120911421-0cb4a180-c6a5-11eb-95b3-b12e5fca15fa.png">
<img width="986" alt="Screenshot 2021-06-06 at 8 55 49 AM" src="https://user-images.githubusercontent.com/2174170/120911426-0faf9200-c6a5-11eb-9f8b-1d91392b3de1.png">

### What did you expect?
`IsRepeatable bool` should be available in introspection.go

### Minimal graphql.schema and models to reproduce
https://github.com/khanakia/gqlgen_chatroom/blob/master/.gqlgen.yml

### versions
 - `gqlgen version: 0.13.0`
 - `go version: 1.16.3`
 - dep or go modules: 99 Designs

**Comment by neoreids** [comment]
this happened to me too

**Comment by khanakia** [comment]
@lwc any update on the issue ?

**Comment by dannypaul** [comment]
I think it’s related to this -https://github.com/99designs/gqlgen/issues/1543#issuecomment-856553630

Can you try downgrading gqlparser to v2.1.0

**Comment by neoreids** [comment]
> I think it’s related to [#1543 (comment)](https://github.com/99designs/gqlgen/issues/1543#issuecomment-856553630)
> 
> Can you try downgrading gqlparser to v2.1.0?

not, it doesn't work for me.

**Comment by khanakia** [comment]
@dannypaul @neoreids No it's not related to gqlparse please check my issue again carefully. It's issue in the Struct Parameter which is not defined 0.13.0 but that parameter does exists in their MAIN or MASTER repo

**Comment by jufemaiz** [comment]
Umm guys?

**Comment by jufemaiz** [comment]
Erm … peeps?

**Comment by khanakia** [comment]
@jufemaiz I do not understand what do you mean ?

**Comment by jufemaiz** [comment]
Definitely seeing issues regarding this two months on.

Something with the combo of v0.13.0 + v0.2.2 of the gqlparser.

Happy to dive in to try and help fix, but still trying to grok the underlying cause (beyond the `__Directives` function of the interface not being defined)

**Comment by khanakia** [comment]
@jufemaiz you can check the repo here for the error https://github.com/khanakia/gqlgen_chatroom

And these packages version i am using

```
github.com/99designs/gqlgen v0.13.0
github.com/vektah/gqlparser/v2 v2.2.0
```

**Comment by lwc** [comment]
Fixed in v0.14.0
